streamlit_pianoroll/frontend/src/main.ts

```diff
@@ -122,3 +158,23 @@ export function onStreamlitRender(event: Event): void {
   // *noteSequence* in the player is a more complex structure than a sequence of notes
   player.noteSequence = midi_data
 }
+
+function updateProgressSvgHeight() {
+  const pianoRollSvgVisualizer = document.querySelector(
+    "#my-svg"
+  )! as SVGElement
+  const progressBarSVG = document.querySelector("#progress-bar")! as SVGElement
+
+  if (!pianoRollSvgVisualizer || !progressBarSVG) return
+
+  const percentage = 16.5
+  const minHeight = 48 // 3rem if font-size is 16px
+  const maxHeight = 80 // 5rem if font-size is 16px
```

Comment:
```suggestion
  const maxHeight = 220 // 13.75rem if font-size is 16px
```

I think it makes sense for full screen mode to allow more range here :)